siliconcompiler/schema/schema_cfg.py

```diff
@@ -759,31 +759,40 @@ def schema_datasheet(cfg, design='default', name='default', mode='default'):
             standard footprint names or a custom naming methodology used in
             conjunction with 'fileset' names in the output parameter.""")
 
-    # Absolute max voltage
-    scparam(cfg, ['datasheet', design, 'limits', 'voltage', name],
-            sctype='(float,float)',
-            shorthelp="Datasheet: absolute voltage limits",
-            switch="-datasheet_limits_voltage 'design pin <(float,float)>'",
-            example=[
-                "cli: -datasheet_limits_voltage 'mydevice vdd (-0.4,1.1)'",
-                "api: chip.set('datasheet','mydevice','limits','voltage','vdd', (-0.4,1.1))"],
-            schelp="""Device absolute minimum/maximum voltage not to be
-            exceeded, specified on a per pin basis.""")
-
     # Absolute max temperatures
     metrics = {'storagetemp': 'storage temperature limits',
+               'soldertemp': 'solder temperature limits',
                'junctiontemp': 'junction temperature limits'}
 
     for item, val in metrics.items():
         scparam(cfg, ['datasheet', design, 'limits', item],
                 sctype='(float,float)',
                 shorthelp=f"Datasheet: absolute {val}",
-                switch=f"-datasheet_{item} 'design <(float,float)>'",
+                switch=f"-datasheet_limits_{item} 'design <(float,float)>'",
                 example=[
-                    f"cli: -datasheet_{item} 'mydevice (-40,125)'",
+                    f"cli: -datasheet_limits_{item} 'mydevice (-40,125)'",
                     f"api: chip.set('datasheet','mydevice','limits','{item}',(-40,125))"],
                 schelp=f"""Device absolute {val} not to be exceeded.""")
 
+    # Thermal Resistence
+    metrics = {'rja': 'junction to ambient thermal resistence',
+               'rjct': 'junction to case (top) thermal resistence',
+               'rjcb': 'junction to case (bottom)thermal resistence',
+               'rjb': 'junction to board thermal resistence',
+               'tjt': 'junction to top characterization parameter',
+               'tjb': 'junction to bottom characterization parameter'}
+
+    for item, val in metrics.items():
+        scparam(cfg, ['datasheet', design, 'thermal', item],
+                unit='C/W',
+                sctype='float',
+                shorthelp=f"Datasheet: {val}",
+                switch=f"-datasheet_thermal_{item} 'design <float>'",
+                example=[
+                    f"cli: -datasheet_thermal_{item} 'mydevice 30.4'",
+                    f"api: chip.set('datasheet','mydevice','thermal','{item}',30.4)"],
+                schelp=f"""Device {item} specified in C/W.""")
```

Comment:
Units already defined and could be modified causing this help comment to be incorrect =

Reply:
Good point. As long as the auto-do generator outputs units.

Reply:
The doc generation shows the units